raml-parser-2/src/main/java/org/raml/v2/api/RamlModelResult.java

```diff
@@ -263,6 +274,16 @@ public ExampleSpec getExampleSpec()
         return exampleSpec;
     }
 
+    /**
+     * @return the RAML NamedExample v1.0 parsed without errors
```

Comment:
fix javadoc s/NamedExample/DocumentationItme/, also fix the one for resource types though it was there before